array.js

```diff
@@ -0,0 +1,41 @@
+ 
+//Rest parameter
```

Comment:
Can you please provide Java Script Documentation to this function!

Reply:
Resolved